requirements/lite/base.txt

```diff
@@ -2,7 +2,7 @@
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
 numpy>=1.17.2, <1.23.1
-torch>=1.9.*, <=1.13.0
+torch>=1.10.0, <=1.13.0
```

Comment:
AFAIK, `.*` was there on purpose so that the CI uses the most recent patch version within the minor version.
```suggestion
torch>=1.10.*, <=1.13.0
```

Reply:
true, let's add it as a comment for our future us :)